### ISSUE TITLE
Bug in getting POS of proper noun.

### ISSUE THREAD

**Issue by solarmist** [polm/fugashi#13]
It looks like there's a bug when using UniDic and getting the pos of a proper noun.

```
In [18]: from fugashi import Tagger, GenericTagger

In [19]: tagger = Tagger('-Owakati')

In [20]: tokens = tagger.parseToNodeList("むかし丹波の国に稻村屋源助という金持ちの商人が住んでいた。")

In [21]: tokens
Out[21]: [むかし, 丹波, の, 国, に, 稻村, 屋, 源助, と, いう, 金持ち, の, 商人, が, 住ん, で, い, た, 。]

In [22]: tokens[5]
Out[22]: 稻村

In [23]: tokens[5].pos
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
TypeError: __new__() takes from 1 to 27 positional arguments but 28 were given
Exception ignored in: 'fugashi.Node.set_feature'
TypeError: __new__() takes from 1 to 27 positional arguments but 28 were given
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-23-7fc5ef581d7d> in <module>
----> 1 tokens[5].pos

fugashi/fugashi.pyx in fugashi.UnidicNode.pos.__get__()

TypeError: 'NoneType' object is unsliceable```

**Comment by solarmist** [comment]
This also occurs with the example sentences in the fugashi branch for spaCy

```
In [32]: import spacy

In [33]: from spacy.lang.ja.examples import sentences

In [34]: nlp = spacy.blank("ja")

In [35]: docs = nlp.pipe(sentences)

In [36]: [d for d in docs]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
TypeError: __new__() takes from 1 to 27 positional arguments but 28 were given
Exception ignored in: 'fugashi.Node.set_feature'
TypeError: __new__() takes from 1 to 27 positional arguments but 28 were given
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-36-f038abad59fc> in <module>
----> 1 [d for d in docs]

<ipython-input-36-f038abad59fc> in <listcomp>(.0)
----> 1 [d for d in docs]

~/.local/share/virtualenvs/lingua-regina-tools-8HFzoJax/lib/python3.7/site-packages/spacy/language.py in pipe(self, texts, as_tuples, n_threads, batch_size, disable, cleanup, component_cfg, n_process)
    814         original_strings_data = None
    815         nr_seen = 0
--> 816         for doc in docs:
    817             yield doc
    818             if cleanup:

~/.local/share/virtualenvs/lingua-regina-tools-8HFzoJax/lib/python3.7/site-packages/spacy/language.py in <genexpr>(.0)
    799         else:
    800             # if n_process == 1, no processes are forked.
--> 801             docs = (self.make_doc(text) for text in texts)
    802             for pipe in pipes:
    803                 docs = pipe(docs)

~/.local/share/virtualenvs/lingua-regina-tools-8HFzoJax/lib/python3.7/site-packages/spacy/lang/ja/__init__.py in make_doc(self, text)
    117
    118     def make_doc(self, text):
--> 119         return self.tokenizer(text)
    120
    121

~/.local/share/virtualenvs/lingua-regina-tools-8HFzoJax/lib/python3.7/site-packages/spacy/lang/ja/__init__.py in __call__(self, text)
     91         unidic_tags = []
     92         for token, dtoken in zip(doc, dtokens):
---> 93             unidic_tags.append(dtoken.pos)
     94             token.tag_ = resolve_pos(dtoken)
     95

fugashi/fugashi.pyx in fugashi.UnidicNode.pos.__get__()

TypeError: 'NoneType' object is unsliceable```

**Comment by solarmist** [comment]
I'm on 
```mecab@ -> ../Cellar/mecab/0.996```
```mecab-unidic@ -> ../Cellar/mecab-unidic/2.1.2```

**Comment by solarmist** [comment]
```

In [51]: Tagger('-Owakati').parseToNodeList("稻村")[0].feature_raw.split(',')
Out[51]:
['名詞',
 '固有名詞',
 '人名',
 '姓',
 '*',
 '*',
 'イナムラ',
 'イナムラ',
 '稻村',
 'イナムラ',
 '稻村',
 'イナムラ',
 '固',
 '*',
 '*',
 '*',
 '*',
 'イナムラ',
 'イナムラ',
 'イナムラ',
 'イナムラ',
 '*',
 '*',
 '"0',
 '2"',
 '*',
 '*']```

**Comment by solarmist** [comment]
@polm Are you familiar with this behavior?

**Comment by polm** [comment]
Thanks for the report, I have not seen this before.

This is not about proper nouns, it's related to field parsing in the version of Unidic that includes kana accent data. The binary distribution of 2.1.2 includes that data. With the accent data, sometimes multiple accents are possible, and that's annotated with a field like `"0,2"`. Since the field contains a literal comma that breaks parsing. 

I'll work on a fix for this today.

**Comment by polm** [comment]
I have released a fix for this as 0.1.10rc2, let me know if that works for you. 

I'm working on some unrelated changes that shouldn't take much longer, so this should see official release this week.

**Comment by solarmist** [comment]
Looks like its working now
